src/components/FindPark.vue

```diff
@@ -22,10 +23,16 @@
       type="text"
       aria-label="Type your search terms here"
       placeholder="Enter City, State to search"
+      aria-describedby="Enter city or address to search for specific parks near you."
```

Comment:
For lines 26 and 34, could you change the value for `aria-describedby` so that it says something like this:

`Enter city and state to search for dog parks in that area.`

Could you make the same type of change on 34 so that `address` is replaced by `state`?

Reply:
I will make this change as well. This was just filler text based on which updates I thought were coming. Now that we have the branches all merged together, we can better reflect the exact phrasing for what the App is capable of handling.